### PR TITLE
[KO Number] Number Extractor For Coverage Improvement

### PR DESCRIPTION
**Pull request by neudurgeshp**
**PR Includes**

- Number Extractor
- Number Definitions
- Spec (Cases 234)
- Existing: 108
- New: 126

   **New Cases Details**
- Integer (47): Passed-40, Skipped-7
- Negative (10): Passed-4, Skipped-6
- Decimal (9): Passed-9
- Power (2): Passed-2
- Fraction (58): Passed-32, Skipped-26

### CODE REVIEW

File: Specs/Number/Korean/NumberModel.json [hunk 1]
```diff
@@ -2663,7 +2655,7 @@
     "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
     "NotSupported": "dotnet",
-    "Comment": "PendingImplementation",
+    "Comment": "Point value parser need to be refactored",
```
Comment (tellarin) [inline Specs/Number/Korean/NumberModel.json:2658]: Why? And does it current fail extraction?

File: Specs/Number/Korean/NumberModel.json [hunk 2]
```diff
@@ -2911,7 +2883,7 @@
     "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
     "NotSupported": "dotnet",
-    "Comment": "PendingImplementation",
+    "Comment": "As per language expert, input needs to be corrected",
```
Comment (tellarin) [inline Specs/Number/Korean/NumberModel.json:2886]: Can you explain why in the PR description?

File: Specs/Number/Korean/NumberModel.json [hunk 3]
```diff
@@ -3044,7 +3008,7 @@
...
```
Comment (tellarin) [inline Specs/Number/Korean/NumberModel.json:3011]: Same, and so on.

File: Specs/Number/Korean/NumberModel.json (hunk 1 above)
Comment (neudurgeshp) [reply Specs/Number/Korean/NumberModel.json:2658]: There is no issue in extraction but during parsing it throws exception for value 2.12**백** from **GetPointValue** method, where we may need to add RoundNumberMap support as well. 
Here is the line..
**pointValue += scale * Config.**ZeroToNineMap**[c];**

**백 ->100**  is round number that is not part of ZeroToNineMap dictionary.

File: Specs/Number/Korean/NumberModel.json (hunk 2 above)
Comment (neudurgeshp) [reply Specs/Number/Korean/NumberModel.json:2886]: Sure.

File: Specs/Number/Korean/NumberModel.json (hunk 3 above)
Comment (neudurgeshp) [reply Specs/Number/Korean/NumberModel.json:3011]: Sure.

**Pull request merged**